### PR TITLE
Table Component

### PR DESCRIPTION
**Pull request by dangalan**
This code is basically copied from [Chakra's docs on integrating React Table](https://chakra-ui.com/guides/integrations/with-react-table).

Currently their are no props styling the table but we can add that once we get some design direction.

### CODE REVIEW

File: components/Table.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+export const Table = ({columns, data}: TableProps ): JSX.Element =>  {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
```
Comment (dangalan) [inline components/Table.tsx:27]: This code is code is basically copied from [Chakra's docs on integrating React Table](https://chakra-ui.com/guides/integrations/with-react-table).

File: components/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    columns,
+    data,
+  })
+
+  return (
+    <ChakraTable {...getTableProps()}>
```
Comment (dangalan) [inline components/Table.tsx:33]: According to the [React Table's docs](https://react-table.tanstack.com/docs/api/useTable#instance-properties), we need to use different functions returned by the `useTable()` hook to resolve props for different components within the table (the overall table wrapper, rows, headers, etc.).

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Mutual Aid NYC Services List
 A [Next.js](https://nextjs.org) website for displaying NYC health, human and social services lists in the [Open Referral](https://openreferral.org) data format. Data is managed via [Airtable](https://airtable.com).
 
-## Tecnical Overview
```
Comment (btmoore4) [inline README.md:4]: Lol good catch

**Review by btmoore4 — APPROVED** [review]
LGTM. Next step I guess will be replace NextJS landing page with an example component

**Pull request merged**